docs/getting-started/integrating-codacy-with-your-git-workflow.md

```diff
@@ -0,0 +1,65 @@
+# Integrating Codacy with your Git workflow
+
+<!--TODO
+    - Review the intro and consider integrating the remaining info gathered from previous versions
+
+Depending on factors such as the current code quality of your repository and the maturity of your team practices, consider the balance between implementing stricter quality gates and the possibility of delaying or blocking the development progress.
```

Comment:
I'd consider dropping this sentence. We mention it further along in a tip admonition under point 1

Reply:
I agree that this is too much detail for the intro. :+1:

However, there's some extra info here that would complement the tip below really well:

- The tip makes a big jump between "the defaults help maintain the current code quality of your repository" and "focus on stopping new critical issues from entering your code base".
- This sentence provides the extra context necessary to understand why it's unadvisable for customers that don't have good enough code quality in their repositories or mature team practices yet to try to follow such strict quality gates. It would basically help glue together the two paragraphs in the tip. :nerd_face:

Reply:
Agreed. I integrated the sentence midway and polished for flow